source/base/logstream.cc

```diff
@@ -414,8 +415,8 @@ LogStream::get_prefixes() const
 void
 LogStream::print_line_head()
 {
-  const std::string &head   = get_prefix();
-  const unsigned int thread = Threads::this_thread_id();
+  const std::string &   head   = get_prefix();
+  const std::thread::id thread = std::this_thread::get_id();
```

Comment:
I bet that this is probably exactly the same number...

Reply:
Yes, probably.